.github/clean.rb

```diff
@@ -0,0 +1,16 @@
+require "fileutils"
+
+ALLOW_LIST = [
```

Comment:
What about renaming this to be `WHITELIST`?

Reply:
I'll dm you.